x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/transforms/DataFrameTransformTask.java

```diff
@@ -207,6 +207,10 @@ public synchronized void start(ActionListener<Response> listener) {
         persistStateToClusterState(state, ActionListener.wrap(
             task -> {
                 auditor.info(transform.getId(), "Updated state to [" + state.getTaskState() + "]");
+                long now = System.currentTimeMillis();
+                // kick off the indexer
+                triggered(new Event(schedulerJobName(), now, now));
```

Comment:
It seems to me, once we add a tad more logic to the `SchedulerEngine.Schedule` this won't be necessary because it will know if it has to be triggered right now, later, or not at all. 

But this is a good stop-gap until we fix the created schedule.